target/linux/generic/files/drivers/net/phy/rtl8366_smi.h

```diff
@@ -23,6 +23,25 @@ struct dentry;
 struct inode;
 struct file;
 
+#define RTL8367B_FAMILY_SHIFT	4
+
+enum {
+	RTL8367B_FAMILY_UNKNOWN,
+	RTL8367B_FAMILY_B_RLVID0,
+	RTL8367B_FAMILY_B_RLVID1,
+	RTL8367B_FAMILY_C
+};
+
+typedef enum rtl8367b_chip_e {
+	RTL8367B_CHIP_UNKNOWN = RTL8367B_FAMILY_UNKNOWN << RTL8367B_FAMILY_SHIFT,
+	RTL8367B_CHIP_UNKNOWN_RLVID0 = RTL8367B_FAMILY_B_RLVID0 << RTL8367B_FAMILY_SHIFT,
+	RTL8367B_CHIP_UNKNOWN_RLVID1 = RTL8367B_FAMILY_B_RLVID1 << RTL8367B_FAMILY_SHIFT, 
```

Comment:
I do not like that the enum uses these strange values. 
I would prefer when the enum would have some normal numbers and is not used with shifts. 

When I see this code:
```
switch (smi->rtl8367b_chip >> RTL8367B_FAMILY_SHIFT) {
```
Maybe just store the family here.

Reply:
I simplified the code, I no longer use RTL8367B_FAMILY_SHIFT.